backend/eduschola_project/edusch_app/serializers.py

```diff
@@ -1,39 +1,47 @@
 from rest_framework import serializers
-from .models import Student, Parent, Instructor, User, School
+
+from django.contrib.auth.models import Group
+from .models import Student, Parent, Staff, User, School, Grade
 
 
 class UserSerializer(serializers.ModelSerializer):
+    # groups = serializers.StringRelatedField(many=True, read_only=True)
     class Meta:
         model = User
-        fields = '__all__'
-
 
-class StudentSerializer(serializers.ModelSerializer):
-    user = UserSerializer()
+        fields = ['username', 'password', 'role', 'school', 'first_name', 'last_name']
```

Comment:
Dont think their username needs to be serialized too, if we already have first and last name